src/UI/pages/index.tsx

```diff
@@ -1,12 +1,26 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from 'next/head';
+import styles from '../styles/Home.module.css';
 import dynamic from 'next/dynamic';
-import ClientOnly from '../components/clientOnly';
+import ClientOnly from '../components/shared/clientOnly';
+import Footer from '../components/shared/footer';
+import { useAppSelector, useAppDispatch } from '../state/hooks';
+import { useEffect } from 'react';
+import { getFeatureFlags } from '../state/configSlice';
 
 const MapComponent = dynamic(() => import("../components/map"), { ssr: false });
 
 function Home(): JSX.Element {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(getFeatureFlags());
```

Comment:
This might be better in _app.tsx so it's triggered as soon as the app loads, it would just be a call to `store.dispatch(getFeatureFlags())`